providers/common/provider_util.c

```diff
@@ -19,6 +19,7 @@
 #endif
 #include "prov/provider_util.h"
 #include "internal/nelem.h"
+#include "crypto/evp.h"
```

Comment:
The "fips change" is quite surprising since all the changes below are contained within "ifndef FIPS_MODULE" - but I suppose it is this include that does it. Could we move it inside the "ifndef FIPS_MODULE" above?

Reply:
Good point